cmd/commands/runtime.go

```diff
@@ -817,7 +828,8 @@ func runRuntimeUpgrade(ctx context.Context, opts *RuntimeUpgradeOptions) error {
 	handleCliStep(reporter.UpgradePhaseStart, "Runtime upgrade phase started", nil, false, true)
 
 	log.G(ctx).Info("Downloading runtime definition")
-	newRt, err := runtime.Download(opts.Version, opts.RuntimeName)
+
+	newRt, err := runtime.Download(opts.Version, opts.RuntimeName, opts.installFeatures)
```

Comment:
when downloading runtime manifest from `runtime.yaml` - send the installFeatures for component filtering